routes/api/v1/favorites.js

```diff
@@ -6,6 +6,47 @@ const configuration = require('../../../knexfile')[environment];
 const database = require('knex')(configuration);
 const fetch = require("node-fetch");
 const env = require('dotenv').config()
+const CurrentForecast = require('../../../current_forecast')
+
+async function fetchCoordinates(location) {
+  let coordinates = await fetch(`https://maps.googleapis.com/maps/api/geocode/json?address=${location},co&key=${process.env.GOOGLE_KEY}`)
```

Comment:
Would the geocoding API be able to find a location outside of Colorado? It looks like you've hardcoded that into the URL.